schema/source.json

```diff
@@ -0,0 +1,96 @@
+        {
+            "title": "description  is required",
+            "required": [
+                "description"
+            ],
+            "description": "A description  is required."
+        },
+        {
+            "title": "scope  is required",
+            "required": [
+                "scope"
+            ],
+            "description": "At least one scope is required."
+        }
+    ],
+    "properties": {
+        "sourceId": {
+            "type": "string",
+            "description": "A unique identifier for the data source.",
+            "translation": {
+                "FGDC CSDGM": [
+                    "dataqual > lineage > srcinfo > srccitea"
+                ]
+            }
+        },
+        "description": {
+            "type": "string",
+            "description": "A brief description about the source dataset used in creating the data identified by the data quality scope.",
+            "translation": {
+                "ISO 19115-2": [
+                    "LI_Source > description"
+                ]
+            }
+        },
+        "sourceCitation": {
+            "description": "A citation providing information about the source dataset, including an online resource or other access instructions.",
+            "translation": {
+                "ISO 19115-2": [
+                    "LI_Source > sourceCitation > CI_Citation"
+                ],
+                "FGDC CSDGM": [
+                    "dataqual > lineage > srcinfo > srccite > citeinfo"
+                ]
+            },
+            "$ref": "citation.json#"
+        },
+        "metadataCitation": {
+            "type": "array",
+            "description": "Citation providing information about the metadata for the source.",
+            "translation": {},
+            "items": {
+                "$ref": "citation.json#"
+            }
+        },
+        "spatialResolution": {
+            "$ref": "./spatialResolution.json#",
+            "description": "The number below the line in a vulgar fraction",
+            "translation": {}
+        },
+        "referenceSystem": {
+            "$ref": "./spatialReference.json#",
+            "description": "Spatial reference system used by the source.",
+            "translation": {}
+        },
+        "sourceProcessStep": {
+            "type": "array",
+            "description": "A description of a non-trivial event or transformation taken to prepare the source data for use in creating the data identified by the data quality scope.",
+            "translation": {
+                "ISO 19115-2": [
+                    "LI_Source > sourceStep > LI_ProcessStep"
+                ]
+            },
+            "items": {
+                "$ref": "#/definitions/processStep"
+            }
+        },
+        "scope": {
+            "$ref": "./scope.json#",
+            "description": "Type of resource and/or extent to which the process information.",
+            "translation": {}
+        }
+    }
```

Comment:
Missing:
processedLevel
resolution

These were clearly not added originally, but they do need to be in there based on the class definition. https://github.com/adiwg/mdJson-schemas/issues/234